apps/discovery_api/lib/discovery_api/application.ex

```diff
@@ -27,8 +27,7 @@ defmodule DiscoveryApi.Application do
         {Brook, Application.get_env(:discovery_api, :brook)},
         DiscoveryApi.Data.CachePopulator,
         supervisor(DiscoveryApiWeb.Endpoint, []),
-        DiscoveryApi.Quantum.Scheduler,
```

Comment:
Pretty much all of the changes from here down are reverting the changes made for the tableau connector endpoint. We are going to fix this too